setuptools/monkey.py

```diff
@@ -138,8 +138,7 @@ def patch_for_msvc_specialized_compiler():
     Patch functions in distutils to use standalone Microsoft Visual C++
     compilers.
     """
-    # import late to avoid circular imports on Python < 3.5
-    msvc = import_module('setuptools.msvc')
+    from . import msvc
```

Comment:
Basically a revert of https://github.com/pypa/setuptools/commit/b7b9cb23f217095e79c618c0e3196712d2d9a285
Unless this still happens in newer versions, in which case the comment should be updated instead.